treadmill-scripts/configureldap.sh

```diff
@@ -115,7 +115,7 @@ trap cleanup EXIT
 # Write userdata to file
 cat << E%O%F > ${tmpdir}/LDAP.yaml
 ---
-treadmill_cell: -
+treadmill_cell: None
```

Comment:
None is not special value in yaml. we interpret --cell - everywhere as cell = None, so - is correct.

Reply:
OK. Have tested and confirmed that YAML will accept a blank space as None, and Python will receive that value as None. Would this be acceptable?